src/main/java/io/stargate/sgv2/jsonapi/api/model/command/impl/CreateCollectionCommand.java

```diff
@@ -54,5 +60,21 @@ public VectorSearchConfig(Integer size, String function) {
         this.function = function == null ? "cosine" : function;
       }
     }
+
+    public record VectorizeConfig(
```

Comment:
I think this class as well as `VectorSearchConfig` should be main level class, not nested in`CreateCollectionCommand` -- since it's not really part of CCC and is used elsewhere.

Also: if this is what is stored in CQL table comment property, I think it should be done in a way that can extended: I suspect we will be adding more configuration properties to persist. So `VectorizeConfig` would be a root-level property of general "StoredCollectionConfig" (or whatever config container is named).